mock-mv/src/test/resources/bootstrap.properties

```diff
@@ -0,0 +1,5 @@
+spring.cloud.config.uri=http://localhost:51000/config
```

Comment:
Test case should work in offline also. Please remove the url and add properties in application.